core/solidus_core.gemspec

```diff
@@ -24,13 +24,13 @@ Gem::Specification.new do |s|
     actionmailer actionpack actionview activejob activemodel activerecord
     activesupport railties
   ].each do |rails_dep|
-    s.add_dependency rails_dep, ['>= 5.1', '< 5.3.x']
+    s.add_dependency rails_dep, ['>= 5.1', '< 7.0.x']
```

Comment:
Knowing from history of a lots of Rails Upgrades we should lower this to 6.1 ;)

Reply:
@tvdeyen you mean because 6.1 might introduce breaking changes? If that's the case, I see your point but I think it would be painful for users who want to upgrade to 6.1 to have to wait for a new Solidus release. On the other hand, locking to 6.0 in case a breaking change is introduced is quite easy.

I don't have a very strong opinion on this though, so I'm happy to lock to < 6.1 if you think a higher upper limit is likely to cause problems.

Reply:
I my experience Rails x.0 upgrades are always the easiest, x.1 updates not necessarily. But we can leave it as is and adjust if necessary.

Reply:
You must not forget that Rails does not follow semver.